### PR TITLE
BETA: Register soothe.is-a.dev

### PR DESCRIPTION
**Pull request by lrmn7**
Added `soothe.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/note-lrmn.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmnid@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "github.com/lrmn7/blogs"
```
Comment (WilliamDavidHarrison) [inline domains/note-lrmn.json:8]:
```suggestion
        "URL": "https://github.com/lrmn7/blogs"
```

File: domains/soothe.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Soothe.is-a.dev",
+    "repo": "https://github.com/soothe-bot/soothe-bot.github.io",
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmn.dev@gmail.com",
+        "discord": "romanromannya#0"
```
Comment (WilliamDavidHarrison) [inline domains/soothe.json:7]:
```suggestion
        "discord": "romanromannya"
```

File: domains/miu.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Miu.is-a.dev",
+    "repo": "https://github.com/Miu-Discord/Miu-Discord.github.io",
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmn.dev@gmail.com",
+        "discord": "romanromannya#0"
```
Comment (WilliamDavidHarrison) [inline domains/miu.json:7]:
```suggestion
        "discord": "romanromannya"
```

File: domains/lucy.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Lucy.is-a.dev",
+    "repo": "https://github.com/Hai-Lucy/Hai-Lucy.github.io",
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmn.dev@gmail.com",
+        "discord": "romanromannya#0"
```
Comment (WilliamDavidHarrison) [inline domains/lucy.json:7]:
```suggestion
        "discord": "romanromannya"
```

File: domains/ohmycv.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmnid@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "https://ohmycv.netlify.app"
```
Comment (WilliamDavidHarrison) [inline domains/ohmycv.json:8]:
```suggestion
        "URL": "https://ohmycv.netlify.app"
```

File: domains/ohmycv.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmnid@gmail.com"
+    },
+
+    "record": {
+        "URL": "https://ohmycv.netlify.app"
```
Comment (WilliamDavidHarrison) [inline domains/ohmycv.json:8]: I cannot reach this website.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/ohmycv.json (hunk 6 above)
Comment (lrmn7) [reply domains/ohmycv.json:8]: The website has been unpublished, please delete it ohmycv.json

File: domains/note-lrmn.json (hunk 1 above)
Comment (lrmn7) [reply domains/note-lrmn.json]: The website has been unpublished, please delete it

**Review by wdhdev — APPROVED** [review]

**Pull request merged**